terraform/testcases/hostmetrics_receiver/otconfig.tpl

```diff
@@ -0,0 +1,76 @@
+
+receivers:
+  hostmetrics:
+    collection_interval: 15s
+    scrapers:
+      cpu:
+        metrics:
+          system.cpu.utilization:
+            enabled: true
+      load:
+      memory:
+        metrics:
+          system.memory.utilization:
+            enabled: true
+      disk:
+      filesystem:
+        metrics:
+          system.filesystem.utilization:
+            enabled: true
+      network:
+        metrics:
+          system.network.conntrack.count:
+            enabled: true
+          system.network.conntrack.max:
+            enabled: true
+      paging:
+        metrics:
+          system.paging.utilization:
+            enabled: true
+      processes:
+      process:
+        metrics:
+          process.memory.physical_usage:
+            enabled: false
+          process.memory.virtual_usage:
+            enabled: false
+          process.memory.usage:
+            enabled: true
+          process.memory.virtual:
+            enabled: true
+          process.context_switches:
+            enabled: true
+          process.cpu.utilization:
+            enabled: true
+          process.memory.utilization:
+            enabled: true
+          process.open_file_descriptors:
+            enabled: true
+          process.paging.faults:
+            enabled: true
+          process.signals_pending:
+            enabled: true
+          process.threads:
+            enabled: true
+
+processors:
+  resourcedetection/ec2:
+    detectors: [ec2]
+    timeout: 2s
+    attributes: [host.id]
+    override: true
+
+exporters:
+  awsemf:
+    namespace: '${testing_id}'
```

Comment:
Is it possible to add a prefix `hostmetrics-test` or something like that so that it is possible to still explore the metrics in the cloudwatch console of the integration test account?
E.g.:
```
namespace: 'hostmetrics-test/${testing_id}'
```